### PR TITLE
fix prop types for ChipsInput props

### PR DESCRIPTION
**Pull request by Dor256**
## Description
*Enter description to help the reviewer understand what's the change about...*
Fix typing of `ChipsInputProps` by extending `TextInputProps`

## Changelog
*Add a quick message for our users about this change*
Fix `ChipsInput` prop typings

### CODE REVIEW

File: typings/components/ChipsInput.d.ts [hunk 1]
```diff
@@ -1,7 +1,9 @@
+import { TypographyModifiers } from 'generatedTypes';
```
Comment (M-i-k-e-l) [inline typings/components/ChipsInput.d.ts:1]: Looks like you've left this be mistake

File: typings/components/ChipsInput.d.ts [hunk 2]
```diff
@@ -1,7 +1,9 @@
...
 import {ReactElement} from 'react';
 import {StyleProp, TextStyle, ViewStyle, AccessibilityProps, ColorValue} from 'react-native';
 import {BaseComponent} from '../commons';
 import {AccessibilityProps} from '../modifiers';
+import { TextInputProps } from './Inputs';
```
Comment (M-i-k-e-l) [inline typings/components/ChipsInput.d.ts:6]: Shouldn't this be `TextFieldProps`?

**Review by M-i-k-e-l — CHANGES_REQUESTED** [review]

File: typings/components/ChipsInput.d.ts (hunk 1 above)
Comment (Dor256) [reply typings/components/ChipsInput.d.ts:1]: woops - fixing

File: typings/components/ChipsInput.d.ts (hunk 2 above)
Comment (Dor256) [reply typings/components/ChipsInput.d.ts:6]: Hmm, I don't really know as I've only seen this used in our module as `TextInput` you know the use cases better than me :)

Comment (Dor256) [reply typings/components/ChipsInput.d.ts]: Fixed

Comment (M-i-k-e-l) [reply typings/components/ChipsInput.d.ts]: > you know the use cases better than me :)
Not really, it's just using a `TextField` with `{...others}` :)

**Review by M-i-k-e-l — APPROVED** [review]

**Pull request merged**